index.rst

```diff
@@ -44,14 +44,636 @@
+The camera image visualization tool is designed to run on the camera diagnostic cluster and to deliver full-resolution focal plane images to the operator within a few seconds of image acquisition. 
+The front-end is web based and supports natural zooming and panning, including zooming to full pixel resolution, as well as the ability to display pixel coordinates and raw pixel value by moving the mouse over the image. 
+It supports a simple but extensible set of image processing algorithms which have been found to be useful for viewing images during camera testing.
+
+The tool uses the `International Image Interchange Framework <https://iiif.io/>`_ (IIIF) with the front-end based on the open source `OpenSeadragon <https://openseadragon.github.io/>`_ framework, and the backend based on the open source `Cantaloupe <https://cantaloupe-project.github.io/>`_ product, modified to be able to read images from  FITS files and directly from the DAQ.
+The tool is currently deployed at SLAC for camera construction, and on the summit in Chile for Auxtel and ComCam.
+
+
+Engineering Facilities Database and Large File Annex
+'''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+The `Engineering Facilities Database (EFD) <https://sqr-034.lsst.io/#introduction>`_ records all commands, events, and telemetry sent over the DDS control network.
+This content essentially tracks the observatory state as a function of time and is very useful in diagnosing issues and understanding (both desired and undesired) operational behaviours.
+The database is best queried using the `EFD client <https://efd-client.lsst.io/>`_ from the `Nublado (Jupyter Notebook) Interface`_ (or any python-based method/script) when making custom plots.
+Instructions on how to access the EFD, and specifically the other instances of the data, are found in `SQR-034 <https://sqr-034.lsst.io/#efd-deployments>`_.
+However, the `Chronograf`_ graphical front-end offers a nice solution for building simple plots (dashboards).
+
+The `Large File Annex (LFA) <https://tstn-029.lsst.io/>`_ contains files over ~1 MB that are accessible both from the summit and the `RSP`_.
+When a file is published to the LFA its presence (and location of the file) is published via SAL/DDS and therefore the location of LFA files can be found via an EFD query.
+It is anticipated that artifacts generated from automated on-the-fly analyses will be stored in this area.
+An example of this would be the `Papermill Executed Parameterized Notebooks`_.
+
+
+Chronograf
+''''''''''
+
+The `summit-based Chronograf interface <https://chronograf-summit-efd.lsst.codes/>`_ (VPN required) provides a user-friendly graphical interface to the each `deployed instance of the EFD <https://sqr-034.lsst.io/#efd-deployments>`_.
+It is particularily useful for creating visualization dashboards to show the current status of the observatory when the LOVE functionality is either not yet functional or simply not planned to be implemented.
```

Comment:
particularily -> particularly 

suggest edit;

> It is particularly useful for quicklook analysis and for creating dashboards to show a particular set of information about the status of the observatory, especially when a similar functionality is either not yet implemented or simply not planned to be implemented in LOVE.

Reply:
rephrased... but didn't take this exact suggestion.